2repository/7.0/community-7.0.0/overload.gradle

```diff
@@ -0,0 +1,22 @@
+ext {
+    alfresco = [
+            version: [
+                    major: 7,
+                    minor: 0,
+                    rev: '0'
+            ],
+            flavor: 'community'
+    ]
+    solrimage = 'docker.io/xenit/alfresco-solr6:1.4.0'
+    shareimage = 'docker.io/xenit/alfresco-share-community:7.0.0'
+    tests = true
+}
+
+dependencies {
+    alfrescoAmp 'org.alfresco:alfresco-share-services:7.0.0@amp'
+    // content-services-community version based on:
+    // https://github.com/Alfresco/acs-community-packaging/blob/7.0.0/pom.xml#L16
```

Comment:
Can you pull the correct version from the POM?

`baseAlfrescoWar platform('org.alfresco:acs-community-packaging:7.0.0')`,
and then put `baseAlfrescoWar 'org.alfresco:content-services-community@war'` might work because the version is pulled from the platform